LiskHQ/lisk
# Remove duplicate asset rows from trs tables - Closes #2054

### What was the problem?
Due to some previous bug in block save. Votes, Dapps, and Multisignature tables had duplicate entries and the tables also did not have unique constraint.
### How did I fix it?
Removed duplicate rows, and added a PRIMARY KEY constraint.
### How to test it?
Load testnet [with snapshot which contains duplicate entries](http://snapshots.lisk.io.s3-eu-west-1.amazonaws.com/lisk/test/lisk_test_backup-4555600.gz), and start the application. The duplicate entries should be removed by added migrations.
### Review checklist

* The PR resolves #2054 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
